tools/eslint/import-resolver.js

```diff
@@ -15,17 +11,5 @@ exports.resolve = function ( source, file, config ) {
 			extensions: [ '.tsx', '.ts', '.mjs', '.js', '.json', '.node' ],
 		} );
 
```

Comment:
This was causing import / linter issues for me, and removing it seemed to resolve them. Is this still needed?